app/controllers/entities_controller.rb

```diff
@@ -42,6 +42,10 @@ def entity
     end
   end
 
+  def entity_caching_enabled
```

Comment:
Let's name this `entity_caching_enabled?` (or would that be a problem for the Mustache template...?)

Reply:
it doesn't work well with mustache, but since it's assigning @entity_caching_enabled it's fine as long as the method is always called before the view is rendered